frontend/src/pages/Confirmation/reducer.js

```diff
@@ -0,0 +1,178 @@
+import { fromJS } from "immutable";
+import { LOGOUT } from "../Login/actions";
+import {
+  FETCH_PROJECT_PERMISSIONS,
+  FETCH_PROJECT_PERMISSIONS_SUCCESS,
+  GRANT_PERMISSION_SUCCESS,
+  REVOKE_PERMISSION_SUCCESS,
+  FETCH_PROJECT_PERMISSIONS_FAILURE
+} from "../Overview/actions";
+import {
+  ASSIGN_PROJECT_SUCCESS,
+  FETCH_SUBPROJECT_PERMISSIONS,
+  FETCH_SUBPROJECT_PERMISSIONS_SUCCESS,
+  GRANT_SUBPROJECT_PERMISSION_SUCCESS,
+  REVOKE_SUBPROJECT_PERMISSION_SUCCESS,
+  FETCH_SUBPROJECT_PERMISSIONS_FAILURE
+} from "../SubProjects/actions";
+import {
+  ASSIGN_SUBPROJECT_SUCCESS,
+  ASSIGN_WORKFLOWITEM_SUCCESS,
+  FETCH_WORKFLOWITEM_PERMISSIONS,
+  FETCH_WORKFLOWITEM_PERMISSIONS_SUCCESS,
+  GRANT_WORKFLOWITEM_PERMISSION_SUCCESS,
+  REVOKE_WORKFLOWITEM_PERMISSION_SUCCESS,
+  FETCH_WORKFLOWITEM_PERMISSIONS_FAILURE
+} from "../Workflows/actions";
+import {
+  ACTION_UPDATE_REQUIRED,
+  CONFIRMATION_CANCELED,
+  CONFIRMATION_CONFIRMED,
+  CONFIRMATION_REQUIRED,
+  EXECUTE_CONFIRMED_ACTIONS,
+  EXECUTE_CONFIRMED_ACTIONS_SUCCESS,
+  STORE_ACTIONS,
+  EXECUTE_CONFIRMED_ACTIONS_FAILURE,
+  STORE_REQUESTED_PERMISSIONS
+} from "./actions";
+
+// original Actions = intents the user has actually requested
+// additional Actions = view permission intents which are required to execute all original actions
```

Comment:
yes, the listPermissions-intent is described as view permissions intent in the ui. The definition as intents is:
- list permissions for getting all permissions information of a project/subproject/wfitem
- view permissions for getting all other information (=intents viewSummary/viewDetails/view in case of workflowitem)
I add it to the comment